src/main/java/Duke.java

```diff
@@ -1,10 +1,97 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner scanner = new Scanner(System.in);
+        Task[] list = new Task[100];
+        int count = 0;
```

Comment:
Perhaps the variable names should be renamed specifically to task in order to avoid confusion in the future.

```suggestion
        Task[] tasks = new Task[100];
        int tasksCount = 0;
```